incubator/node-red/README.md

```diff
@@ -0,0 +1,86 @@
+# Node-RED Appsody Stack
+
+This is a stack for building a deployable Node-RED application using [Appsody](https://appsody.dev).
```

Comment:
Could you mention here that `This stack is based on the `Node.js v10` runtime and allows you to build and deploy a new or existing Node-RED application using Appsody....